WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -443,7 +447,8 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
                             List<IEdmProperty> loadedProperties = new List<IEdmProperty>();
                             foreach (PropertyInfo property in ClientTypeUtil.GetPropertiesOnType(type, /*declaredOnly*/edmBaseType != null).OrderBy(p => p.Name))
                             {
-                                IEdmProperty edmProperty = this.CreateEdmProperty(complexType, property);
+                                FieldInfo backingField = FindBackingField(property, this.ProxyBackingFieldNamingConvention);
```

Comment:
No, Complex properties have the form:
`        public Som.Permission.Read AccessCode
        {
            get
            {
                if (((this._AccessCode == null) 
                            && (this._AccessCodeInitialized != true)))
                {
                    this._AccessCode = new Som.Permission.Read();
                    this._AccessCodeInitialized = true;
                }
                return this._AccessCode;
            }
            set
            {
                this.OnAccessCodeChanging(value);
                this._AccessCode = value;
                this._AccessCodeInitialized = true;
                this.OnAccessCodeChanged();
                this.OnPropertyChanged("AccessCode");
            }
        }
        private Som.Permission.Read _AccessCode;
        private bool _AccessCodeInitialized;
`